Gruntfile.js

```diff
@@ -139,6 +139,9 @@ module.exports = function(grunt) {
       bower_update: {
         cmd: 'bower update'
       },
+      pip_install: {
```

Comment:
This is not related to the functional changes in the PR, please revert (also the changes to grunt.registerTask below)

In fact, doing a pip install should not be necessary for using the dev version of the theme with the docs.

Reply:
I agree, that it does not related to the functionality of the PR, so I revert it. 

For me my changes were not effective until a pip install --upgrade . , but maybe it is something with my environment.